Gds/setup.py

```diff
@@ -33,6 +33,8 @@
 from setuptools import find_packages
 from setuptools import setup
 
+import src.fastentrypoints
```

Comment:
Why is this imported as src.fastentrypoints?

Reply:
Are you running pip install Gds/ or python setup.py install or both?  Does this file work for both?

Reply:
Hey Michael, there wasn't any particular reason I did this; that's just where I put this script and this was how I knew to import it. Both methods work for me if I run this command from within the Gds directory. If I run it from the root, `pip install Gds/.` works fine, but `python Gds/setup.py install` fails.

Weirdly, even if I completely remove this line, calling `python Gds/setup.py install` fails with the following message until I run it from within the Gds folder:

```bash
(fprime-venv) vagrant@vagrant:~/jpl/fprime$ python Gds/setup.py install
running install
running bdist_egg
error: error in 'egg_base' option: 'src' does not exist or is not a directory
```